A/Aeron/build_tarballs.jl

```diff
@@ -63,4 +78,7 @@ dependencies = [
 ]
 
 # Build the tarballs, and possibly a `build.jl` as well.
-build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; julia_compat="1.6", preferred_gcc_version = v"11.1.0")
+build_tarballs(
+    ARGS, name, version, sources, script, platforms, products, dependencies;
+    julia_compat="1.6", preferred_gcc_version = v"11.1.0")
```

Comment:
This is a C++ library, so if you use GCC 11 it may not work in practice with Julia v1.6 (depending on the `GLIBCXX` symbols used): https://github.com/JuliaPackaging/Yggdrasil/blob/5a60de3a72c2a07a2ddd74781a12e6173a279c39/CONTRIBUTING.md#compatibility-tips

Reply:
Thanks! 
I think we are actually using only the c library, but it's obvious how to turn off the C++ build. It always wants to build both.

Regardless I'll try without the preferred gcc version.

Reply:
I removed the preferred_gcc_version.